### DISCUSSION TITLE
use the app for english (translated) manga, how to setup?

### DISCUSSION THREAD

**Discussion by Daoet** [arianneorpilla/jidoujisho#353] [Q&A]
So I read JidouJisho has support for the english language.
Assuming a Monolingual setup (english as target language), how would you set this up for manga that is already in english, particularly the mokuro part?

I'd like to use the app to read english (translated) manga, and need some guidance on how to set this up. As I understand it, mokuro supports Japanese manga.

**Comment by brishtibheja** [comment]
I don't think mokuro supports English. But just to be sure you can ask a question on the mokuro fork of ZXY.
